pkgs/tools/misc/fdupes/default.nix

```
@@ -1,17 +1,21 @@
-{ stdenv, fetchFromGitHub }:
+{ stdenv, fetchFromGitHub, autoconf, automake, ncurses, pcre2, autoreconfHook }:
```

```suggestion
{ stdenv, fetchFromGitHub, autoreconfHook, ncurses, pcre2 }:
```

With `autoreconfHook` you can remove `autoconf` and `automake`. I would also order `autoreconfHook` before `ncurses` and `pcre`.

---

Fixed, thanks